src/mame/includes/pgm2.h

```diff
@@ -42,17 +42,16 @@ class pgm2_state : public driver_device
 		m_bgscroll(*this, "bgscroll"),
 		m_fgscroll(*this, "fgscroll"),
 		m_vidmode(*this, "vidmode"),
-		m_gfxdecode2(*this, "gfxdecode2"),
-		m_gfxdecode3(*this, "gfxdecode3"),
+		m_gfxdecode(*this, "gfxdecode%u", 2U),
 		m_arm_aic(*this, "arm_aic"),
 		m_sprites_mask(*this, "sprites_mask"),
 		m_sprites_colour(*this, "sprites_colour"),
 		m_sp_palette(*this, "sp_palette"),
 		m_bg_palette(*this, "bg_palette"),
 		m_tx_palette(*this, "tx_palette"),
 		m_mcu_timer(*this, "mcu_timer"),
-		m_memcard(*this, "memcard_p%u", 1U)
-	{ }
+		m_maindata(*this, "maindata"),
+		m_memcard(*this, "memcard_p%u", 1U) { }
```

Comment:
Please don't put the `{ }` on the end of a line when the initialiser list is split across multiple lines.